pkg/gds/certman/certs_test.go

```diff
@@ -205,6 +205,166 @@ func (s *certTestSuite) TestCertManager() {
 	require.Equal("automated", certReq.AuditLog[5].Source)
 }
 
+func (s *certTestSuite) TestCertManagerThirtyDayReissuanceReminder() {
+	_ = s.setupCertManager(sectigo.ProfileCipherTraceEE)
+	defer s.teardownCertManager()
+	defer s.fixtures.LoadReferenceFixtures()
+	require := s.Require()
+
+	// setup the datastore to contain the modified echoVASP
+	s.resetVASPDatabase()
```

Comment:
You wouldn't have known this because you haven't had to deal with the GDS fixtures, but our strategy throughout the GDS tests is that we rely on a set of "source of truth" VASP and certreq fixtures which are compressed into an archived file and included in the repo (at pkg/gds/testdata/fakes.tgz). Therefore, we generally prefer to use the existing fixtures rather than hacking our own inside of the tests (although sometimes we do have to modify them at test time). My strategy for writing tests that require fixtures is to check if there are any existing fixtures that are in the required state and if not, modify as necessary in the tests.

For this specific situation, I believe the requirements are that we need a verified VASP with verified administrative and technical contacts? In that case, it looks like `hotel` is a good fit, although it looks like you still need to set the administrative and technical contacts to something verified for your tests.

My suggestion is to use that VASP fixture and remove `resetVASPDatabase` because `teardownCertManager` will take care of that.

Reply:
Looking through the `fakes.tgz` file, I can't find any VASP fixtures that had both a verified administrative and technical contact. On `hotel` the technical contact is verified but the administrative contact is null.

Additionally, `resetVASPDatabase` is removing the superfluous VASPs from the database because `HandleCertificateReissuance` is using `db.ListVASPs` which means we would have to track the sent emails of the 10+ VASPs already in the database and all of their contacts.